mobile/did_key.go

```diff
@@ -0,0 +1,140 @@
+package mobile
+
+import (
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/crypto/jwx"
+	"github.com/TBD54566975/ssi-sdk/did/key"
+	"github.com/goccy/go-json"
+	"github.com/sirupsen/logrus"
+)
+
+// GetSupportedKeyTypes returns a list of supported key types as string values
+func GetSupportedKeyTypes() []string {
+	keyTypes := make([]string, 0, len(key.GetSupportedDIDKeyTypes()))
+	for _, kt := range key.GetSupportedDIDKeyTypes() {
+		keyTypes = append(keyTypes, string(kt))
+	}
+	return keyTypes
+}
+
+type GenerateDIDKeyResult struct {
+	DID string         `json:"did"`
+	JWK map[string]any `json:"jwk"`
+}
+
+// GenerateDIDKey generates a new DID key and returns a JSON representation of GenerateDIDKeyResult
+func GenerateDIDKey(kt string) ([]byte, error) {
```

Comment:
Can you include the possible kt values in the godoc?

Reply:
I'll add that the values are the result of the GetSupportedDIDKeyTypes